airbyte_cdk/sources/declarative/decoders/composite_raw_decoder.py

```diff
@@ -42,6 +42,30 @@ def parse(
             yield from self.inner_parser.parse(gzipobj)
 
 
+@dataclass
+class JsonParser(Parser):
+    """
+    Parser strategy for converting JSON-structure str, bytes, or bytearray data into MutableMapping[str, Any].
+    """
+
+    encoding: str = "utf-8"
+
+    def parse(self, data: BufferedIOBase) -> Generator[MutableMapping[str, Any], None, None]:
+        raw_data = data.read()
+        try:
+            body_json = json.loads(raw_data.decode(self.encoding))
```

Comment:
Should we use orjson instead of json? If we do, we should make sure it is not affected by bugs like https://github.com/airbytehq/airbyte-python-cdk/pull/210